test/machineslib.py

```diff
@@ -19,29 +19,34 @@
 import sys
 import time
 import traceback
+import unittest
+from collections.abc import Mapping, Sequence
 
 import netlib
 import storagelib
 import testlib
 from machine import testvm
 
 
-def hasMonolithicDaemon(image):
+def hasMonolithicDaemon(image: str) -> bool:
     return (image.startswith("rhel-8-") or
             image.startswith("debian") or
             image.startswith("ubuntu") or
             image in ["arch"])
 
 
 # software TPM doesn't work on some OSes
-def hasBrokenTPM(image):
+def hasBrokenTPM(image: str) -> bool:
     return image.startswith("rhel-8")
 
 
-class VirtualMachinesCaseHelpers:
+class VirtualMachinesCaseHelpers(unittest.TestCase):
     machine: testvm.Machine
+    browser: testlib.Browser
 
-    def waitPageInit(self):
+    def allow_journal_messages(self, *_patterns: str) -> None: ...
```

Comment:
That's a method of testlib.py. Why does that need to be declared here? Or is this temporary until we pull in a typed testlib.py?